
 Ошибка при поиске без результатов. На странице с результатом поиска находится сообщение "No movies where found.":
======================================================================
ERROR: test_untitled (__main__.Untitled)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:/BARANCEV/Seleniun_php4dvd_lesson2_Rovinsky/php4dvd/php4dvd_search_positive.py", line 33, in test_untitled
    assert element.text == "No movies where found."
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webelement.py", line 71, in text
    return self._execute(Command.GET_ELEMENT_TEXT)['value']
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webelement.py", line 454, in _execute
    return self._parent.execute(command, params)
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 201, in execute
    self.error_handler.check_response(response)
  File "C:\BARANCEV\Seleniun_php4dvd_lesson2_Rovinsky\ENV\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 188, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: Element not found in the cache - perhaps the page has changed since it was looked up


----------------------------------------------------------------------
Ran 1 test in 37.984s

FAILED (errors=1)

Process finished with exit code 1
